_layouts/contractors.html

```diff
@@ -14,11 +14,17 @@
       <ul class="menu nav nav-pills nav-stacked">
         {% for title in site.data.config.titles %}
         <li>
-          <a href="/contractors/{{ title.key }}/" title="{{ site.data.config.valueMap[title.key] }}"{% if subdir == title.key %} class="active"{% endif %}>{{ site.data.config.valueMap[title.key] }}</a>
+          <a href="/contractors/{{ title.key }}/" title="{{ site.data.config.valueMap[title.key] }}"{% if subdir == title.key %} class="active"{% endif %}>
```

Comment:
I changed this on the API side, since this is pulling from our API here-

https://github.com/WPContractors/wpcontractors.github.io/blob/develop/_config.yml#L43

Reply:
(so please remove it)

Reply:
The code was changed.